### DISCUSSION TITLE
Is there any tutorial for multi route in next js

### DISCUSSION THREAD

**Discussion by yashnerkar** [gilbarbara/react-joyride#1044] [General]
I want to implement for multi route.  Is there any resource which I can follow to implement in next js

**Comment by gilbarbara** [comment]
Did you check the multi-route [example](https://codesandbox.io/p/devbox/github/gilbarbara/react-joyride-demo/tree/main/?file=%2Fsrc%2FMultiRoute%2Findex.tsx) in the demo?

**Reply by yashnerkar** [reply]
Yes I have checked it, but I got some errors. So could not implemented properly

**Reply by gilbarbara** [reply]
Which errors? Which next.js version? Which pattern? App or page?
Please be more specific.
